community/networkmanager/APKBUILD

```diff
@@ -2,10 +2,10 @@
 # Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=networkmanager
 pkgver=1.18.1
-pkgrel=1
+pkgrel=2
```

Comment:
Enabling arches does not require a `pkgrel` bump (which would trigger a rebuild of all packages).